content/snowplow-recommendations/_index.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Introduction"
+menuTitle = "Introduction"
+pre = "<i class='fas fa-rocket'></i> "
+chapter = false
+weight = 1
++++
+
+### Recommendations Accelerator
+
+#### Introduction
+
+This accelerator will show you how to use the [AWS Personalize service](https://docs.aws.amazon.com/personalize/latest/dg/what-is-personalize.html) to create recommenders based on your Snowplow data, enabling you to provide personalized recommendations to your users. There are two domain use cases that are covered in this accelerator: E-commerce and Video On Demand. Although the instructions will generally reference both use cases, you can choose to do one or the other, or both.
+
+
+
+In this accelerator, we will enable you to:
+
+* Run dbt to transform your Snowplow data into a format that can be used by AWS Personalize
+* Set up the infrastructure required to run AWS Personalize, including the necessary IAM roles and policies
+* Create ecommerce and/or media (video on demand) recommenders in AWS Personalize
+* Run a local Flask application to interact with AWS Personalize 
+***
```

Comment:
Can you add an image of the app here, having a picture on the front page is always good